src/test/compressor/test_compression.cc

```diff
@@ -330,7 +330,11 @@ INSTANTIATE_TEST_CASE_P(
 #endif
     "zlib/noisal",
     "snappy",
-    "zstd"));
+    "zstd"
+#ifdef HAVE_BROTLI
```

Comment:
nit, could add this line before "zstd", less code churn, and better looking this way =)